### DISCUSSION TITLE
charm.sh - Go libraries for TUI and CLI

### DISCUSSION THREAD

**Discussion by danie-dejager** [dundee/gdu#244] [Ideas]
@dundee Have you had look at the libraries from charm.sh? They are offering libraries that can be used for awesome looking TUI and CLI utilities. 
https://charm.sh/

**Comment by dundee** [comment]
Gdu's TUI is built around https://github.com/rivo/tview and it would be a huge effort to change that... But maybe another project can use Gdu abilities to analyse the disku usage fast and be built around Charm libraries.
